### DISCUSSION TITLE
Failure while trying to build from the command line with build_utm.sh

### DISCUSSION THREAD

**Discussion by gubnota** [utmapp/UTM#6133] [Development]
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Configuration (required)**
* UTM Version: v4.4.5
* OS Version:  - (crashed on building stage)
* Device Model: - (crashed on building stage)
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? Build using build_utm.sh

`➜  UTM git:(main) ./scripts/build_utm.sh -p ios -a arm64 -o ~/Downloads/utm_out`
**Crash log**
```log
} replace_IOServiceAuthorize = { IOServiceAuthorizeReplacement, IOServiceAuthorize };
                                                                ^
In file included from ~/Downloads/UTM/JailbreakInterposer/JailbreakInterposer.c:19:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.2.sdk/System/Library/Frameworks/IOKit.framework/Headers/IOKitLib.h:622:1: note: 'IOServiceAuthorize' has been
      explicitly marked unavailable here
IOServiceAuthorize(
^
1 error generated.

** ARCHIVE FAILED **


The following build commands failed:
	CompileC UTM/Build/Intermediates.noindex/ArchiveIntermediates/iOS/IntermediateBuildFilesPath/UTM.build/Release-iphoneos/JailbreakInterposer.build/Objects-normal/arm64/JailbreakInterposer.o UTM/JailbreakInterposer/JailbreakInterposer.c normal arm64 c com.apple.compilers.llvm.clang.1_0.compiler (in target 'JailbreakInterposer' from project 'UTM') (1 failure)
```

**Comment by osy** [comment]
I think you didn't unpack/build the dependencies correctly.

**Reply by 100tomer** [reply]
Can you explain a bit more? For me I downloaded the artifacts and extract it to the same folder as the "output" folder in the command.

**Reply by gubnota** [reply]
I will try to reproduce the issue
